projects/igniteui-angular/src/lib/core/styles/components/carousel/_carousel-theme.scss

```diff
@@ -303,6 +303,8 @@
         overflow: hidden;
         outline-style: none;
         border-radius: --var($theme, 'border-radius');
+        min-height: 300px;
```

Comment:
I think the min-height should be the height of the navigation buttons. How was 300px picked?

Reply:
I picked it randomly if I make the height the same as the buttons the carousel will look like broken. See the attached image.
[image](https://ibb.co/vXJK9D3)